### PR TITLE
Rename source.amount to source.maxAmount

### PR DESCRIPTION
**Pull request by clark800**

### CODE REVIEW

File: src/api/common/schemas/payment.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "title": "payment",
   "type": "object",
   "properties": {
-    "source": {"$ref": "adjustment"},
+    "source": {"$ref": "maxAdjustment"},
```
Comment (lumberj) [inline src/api/common/schemas/payment.json]: Where is the schema for maxAdjustment?

Comment (clark800) [inline src/api/common/schemas/payment.json]: I must have deleted it on accident, fixing now

**Comment by lumberj** [comment]
LGTM. Do you have any ongoing work to update READMEs with new API?

**Comment by clark800** [comment]
There's this https://ripplelabs.atlassian.net/browse/RLJS-372, but I haven't been actively working on it recently. I'm discussing with Rome how we can document it.

**Pull request merged**